python/ray/experimental/gpu_object_manager/gpu_object_store.py

```diff
@@ -57,42 +66,45 @@ def __ray_send__(self, communicator_name: str, obj_id: str, dst_rank: int):
             raise ValueError(
                 f"tensor device {tensor.device} does not match device {device}"
             )
-        collective.send(tensor, dst_rank, group_name=communicator_name)
+        collective.send(
```

Comment:
For consistency, it's better to also call `send` on the transport manager instead of directly on `collective`.